rtil/src/native/character.rs

```diff
@@ -159,9 +178,12 @@ struct FUniqueNetIdSteam {
 #[rtil_derive::hook_once(AMyCharacter::Tick)]
 fn save(this: usize) {
     CHARACTER.set(this);
-    let my_character = AMyCharacter::get_player();
+    let mut my_character = AMyCharacter::get_player();
```

Comment:
No `mut` needed here with the above changes.

Reply:
Cannot borrow as mut; afaik mut is required.